stan/math/prim/scal/prob/normal_sufficient_lpdf.hpp

```diff
@@ -15,13 +14,14 @@
 #include <stan/math/prim/scal/meta/include_summand.hpp>
 #include <stan/math/prim/scal/meta/VectorBuilder.hpp>
 #include <stan/math/prim/scal/meta/max_size.hpp>
+#include <string>
 
 namespace stan {
 
   namespace math {
 
     /**
-     * The log of the normal density for the specified scalar(s) given
+     * The log of the normal density for the specified scalar(s given
```

Comment:
Yes. Fixing. (I'll walk through all the changes to make sure I didn't mistakenly delete anything else.)